verify/cli.go

```diff
@@ -20,7 +20,7 @@ var (
 
 // TODO: should this live in the resources pkg?
 // So that if we ever change the SRS files there we can change this value
-const srsOrder = 268435456 // 2 ^ 32
+const srsOrder = 268435456 // 2 ^ 28
```

Comment:
```suggestion
const srsOrder = 1 << 28
```

Reply:
Should we also remove `MaxSRSPoints` above and use srsOrder instead (since its hardcoded anyways)?

Reply:
[bc7642e](https://github.com/Layr-Labs/eigenda-proxy/pull/228/commits/bc7642e2944f7f9bdefac29b45c130ba00a68066)